apps/web/Dockerfile

```diff
@@ -64,4 +65,6 @@ ARG SELF_HOSTED
 ENV NEXT_PUBLIC_SELF_HOSTED=$SELF_HOSTED
 ENV HOSTNAME=0.0.0.0
 
+HEALTHCHECK CMD curl --fail http://localhost:3000/api/auth/session || exit 1
```

Comment:
Some notes:
* We probably shouldn't be doing a healthcheck against the session endpoint since it's not its intended purpose.
* We should use the `$PORT` env var incase the value gets overridden.

End result would look something like this with the `api/status` endpoint still to be created.

```
HEALTHCHECK --interval=30s --timeout=5s --start-period=30s --retries=3 \
  CMD curl -f http://localhost:${PORT}/api/status || exit 1
```

Reply:
Editing this !

Reply:
That should be better. 
Once the api/status exists